s2c_sensor_module/src/main.c

```diff
@@ -106,7 +113,61 @@ inline float uint16ToC(uint16_t data) {
 	return (float)data * 0.02 - 273.15;
 }
 
+/*
+Does a long blink on the user led
+Warning: 1000ms delay total
+*/
+void flash_high(){
+	port_pin_set_output_level(LED_USER_PIN, true);
+	delay_ms(600);
+	port_pin_set_output_level(LED_USER_PIN, false);
+	delay_ms(400);
+}
+
+/*
+does a short blink on the user led
+Warning: 500ms delay
+*/
+void flash_low(){
+	port_pin_set_output_level(LED_USER_PIN, true);
+	delay_ms(50);
+	port_pin_set_output_level(LED_USER_PIN, false);
+	delay_ms(450);
+}
+
+/*
+quick blink
+*/
+void quick_blink(){
+	port_pin_toggle_output_level(LED_USER_PIN);
+	delay_ms(30);
+	port_pin_toggle_output_level(LED_USER_PIN);
+	delay_ms(30);
+}
+
+/*
+flashes out the binary of the board ID
+Warning long delay
+*/
+void flash_pinstrap_id(uint8_t pin_num){
+	int blink_amount = 5;
+	for(int i =0; i <blink_amount; i++){
+		quick_blink();
+	}
+	port_pin_set_output_level(LED_USER_PIN, false);
+	delay_ms(1000);
+
+	uint8_t temp_pin = pin_num;
+	for(int i =0; i<4;i++){
```

Comment:
Try to keep format consistent with the rest of the codebase: equal spacing between operators, spacing between open brace and statement/function signature